bundle-workflow/jenkins_workflow/integtest/Jenkinsfile

```diff
@@ -0,0 +1,69 @@
+pipeline {
+    agent none
+    environment {
+        BUNDLE_TEST_ID = "${BUILD_NUMBER}"
+    }
+    tools {
+        jdk "JDK14"
+        maven "maven-3.8.2"
+    }
+    stages {
+        stage('Parameters') {
+            steps {
+                script {
+                    properties([
+                            parameters([
+                                    string(
+                                            defaultValue: '',
+                                            name: 'bundle_manifest',
```

Comment:
Why are both manifest needed?

Reply:
As covered in the description, integration tests needs to understand what maven artifacts it needs to pull and where to pull them from for a given component. 
That said, we agreed upon having a single manifest to feed to test system which will be the source of truth.
Opened up an issue to take care of it: https://github.com/opensearch-project/opensearch-build/issues/433